docs/pages/helm/guides/ownconfig.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Running a Teleport cluster with a custom Teleport configuration in Kubernetes using Helm
+description: How to install and configure a Gravitational Teleport cluster with a custom config in Kubernetes using Helm
+---
+
+In this guide, we'll go through how to set up a Teleport cluster in Kubernetes using a custom [`teleport.yaml`](../../config-reference.mdx)
+config file using Teleport Helm charts.
+
+This setup can be useful when you already have an existing Teleport cluster and would like to start running it in Kubernetes, or when
+migrating your setup from an older version of the Helm charts.
+
+We also have a specific [migration guide from older versions of the Helm charts](./migration.mdx) which may also be useful.
```

Comment:
All exists from the article should be "Next steps". Let's try to minimize the amount of "exit points" in the flow of one article. Let's optimize for "quick scroll" flow.